
geometry_msgs/Point[] locations
MoveOptions options
float32 yaw
